javascript/packages/orchestrator/src/providers/podman/resources/grafanaResource.ts

```diff
@@ -56,8 +56,11 @@ export class GrafanaResource {
         `${this.dataSourcesPath}/prometheus.yml`,
         grafanaConfig,
       );
-    } catch {
-      throw new Error("Error generating config for grafana resource");
```

Comment:
We shouldn't use this pattern, since the error message don't add any extra value about the err.